wagtail/admin/templates/wagtailadmin/shared/side_panels/checks.html

```diff
@@ -1,44 +1,60 @@
 {% load i18n wagtailadmin_tags %}
-<template id="w-a11y-result-row-template">
-    <div class="w-a11y-result__row" data-a11y-result-row>
-        <div>
-            <h3 class="w-a11y-result__header">
-                <span class="w-a11y-result__name" data-a11y-result-name></span>
-            </h3>
-            <div class="w-a11y-result__help" data-a11y-result-help></div>
+{% block result_template %}
+    <template id="w-a11y-result-row-template">
+        <div class="w-a11y-result__row" data-a11y-result-row>
+            <div>
+                <h3 class="w-a11y-result__header">
+                    <span class="w-a11y-result__name" data-a11y-result-name></span>
+                </h3>
+                <div class="w-a11y-result__help" data-a11y-result-help></div>
+            </div>
+            <button
+                class="w-a11y-result__selector"
+                data-a11y-result-selector
+                aria-label="{% trans 'Show issue' %}"
+                type="button"
+            >
+                <span class="w-a11y-result__selector-text" data-a11y-result-selector-text></span>
+                {% icon name="link-external" classname="w-a11y-result__icon" %}
+            </button>
         </div>
-        <button
-            class="w-a11y-result__selector"
-            data-a11y-result-selector
-            aria-label="{% trans 'Show issue' %}"
-            type="button"
-        >
-            <span class="w-a11y-result__selector-text" data-a11y-result-selector-text></span>
-            {% icon name="link-external" classname="w-a11y-result__icon" %}
-        </button>
-    </div>
-</template>
-<div class="w-divide-y w-divide-border-furniture w-py-6 w-pl-2 lg:w-pl-8">
-    <div>
-        <h2 class="w-my-5 w-text-16 w-font-bold w-text-text-label">
-            {% trans 'Content metrics' %}
-        </h2>
-        <div class="w-flex w-gap-10">
+    </template>
+{% endblock %}
+
+{% block content %}
+    <div class="w-divide-y w-divide-border-furniture w-py-6 w-pl-2 lg:w-pl-8 w-gap-3">
+        {% block metrics_section %}
             <div>
-                <h3 class="w-my-2 w-text-14 w-text-text-placeholder">{% trans 'Words' %}</h3>
-                <p class="w-font-semibold w-text-text-label" data-content-word-count>-</p>
+                <h2 class="w-my-5 w-text-16 w-font-bold w-text-text-label">
+                    {% trans 'Content metrics' %}
+                </h2>
+                <div class="w-flex w-flex-wrap w-gap-x-10 w-gap-y-4">
```

Comment:
Added `w-flex-wrap` and changed `w-gap-10` into separate x and y gaps so if you add custom metrics it wraps around nicely. The `w-gap-10` is too big of a gap vertically.